Runtimes/Core/CMakeLists.txt

```diff
@@ -171,10 +173,12 @@ add_compile_options(
   "$<$<COMPILE_LANGUAGE:Swift>:SHELL:-enable-experimental-feature Extern>"
   "$<$<COMPILE_LANGUAGE:Swift>:SHELL:-enable-experimental-feature ValueGenerics>"
   "$<$<COMPILE_LANGUAGE:Swift>:SHELL:-enable-experimental-feature AddressableParameters>"
-  "$<$<COMPILE_LANGUAGE:Swift>:SHELL:-Xfrontend -enable-experimental-concise-pound-file>"
+  "$<$<AND:$<PLATFORM_ID:Darwin>,$<COMPILE_LANGUAGE:Swift>>:-save-optimization-record=bitstream>"
```

Comment:
Any reason to restrict this to Darwin?

Reply:
Can you file an issue (and link it in the file) to enable this on other platforms?

Reply:
Filed https://github.com/swiftlang/swift/issues/79279 and mentioned it in a comment.